output/v1/konveyor/violations.go

```diff
@@ -104,6 +150,46 @@ type Dep struct {
 	FileURIPrefix      string                 `json:"prefix,omitempty" yaml:"prefix,omitempty"`
 }
 
+func (d Dep) MarshalYAML() (interface{}, error) {
+	sort.Strings(d.Labels)
+
+	return d, nil
+}
+
+func (d Dep) cmpLess(other Dep) bool {
```

Comment:
What is this used for?

Reply:
I accidentally left it in before committing. I was doing some exploration on both JSON and YAML.